integr8ly/integreatly-operator
# remove rhmiconfig

# Description
Disable functionality of RHMIConfig to stop uninteded upgrades from happening. 

## Verification
1. Build and Install RHMI 2.8.0 to cluster from catalog source
2. set RHMIConfig CR to upgrade scheduled true and set a time in the future when you want the upgrade to happen and notBeforeDays to 0
3. Build and update catalog source for this branch as 2.8.1
4. Verify upgrade happens at configured time 
5. When install complete, repeat steps 2 and 3 as 2.8.2
6. Verify upgrade does NOT happen 
7. When install is complete, repeast steps 2 and 3 as 2.8.3 with the CSV update to be not service affecting and a time much futher in the futre
8. verify upgrade happens automatically, regardless of scheduled time. 

## Type of change




## Checklist
